scripts/restart-server

```diff
@@ -127,8 +127,13 @@ elif action == "start":
 
 def restart_or_start(service: str) -> None:
     our_verb = action
-    if our_verb == "restart" and len(list_supervisor_processes([service], only_running=True)) == 0:
+    existing_services = list_supervisor_processes([service])
+    running_services = list_supervisor_processes([service], only_running=True)
```

Comment:
It's unfortunate that we end up running this twice, since IIRC it takes nonzero time, but I think this is the only way to do this while still supporting the `*` syntax we pass to this function.